org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/xsi/XSIFormatterExperimentalTest.java

```diff
@@ -71,16 +102,47 @@ public void xsiSchemaLocationSplitOnElement() throws BadLocationException {
 				"    xsi:schemaLocation=\"http://www.springframework.org/schema/beans\r\n" + //
 				"                        http://www.springframework.org/schema/beans/spring-beans.xsd\r\n" + //
 				"                        http://www.springframework.org/schema/util\r\n" + //
-				"                        http://www.springframework.org/schema/util/spring-util.xsd\">\r\n" + //
-				"\r\n" + //
-				"</beans>";
-		assertFormat(content, expected, settings);
+				"                        http://www.springframework.org/schema/util/spring-util.xsd\"> </beans>";
+		assertFormat(content, expected, settings,
+				te(5, 24, 6, 8, ""), //
+				te(6, 51, 6, 52, "\r\n                        "), //
+				te(6, 112, 7, 8, "\r\n                        "), //
+				te(7, 50, 7, 51, "\r\n                        "));
+		assertFormat(expected, expected, settings);
+	}
+
+	@Override
+	public String toString() {
+		return "XSIFormatterExperimentalTest []";
 	}
 
-	@Disabled
 	@Test
 	public void xsiSchemaLocationSplitOnPair() throws BadLocationException {
 		SharedSettings settings = createSettings();
+		settings.getFormattingSettings().setSplitAttributes(false);

```

Comment:
I think you should force setPreserveAttributeLineBreak to false because it will become to true by default. I think you should force too for other test splitAttributes to false to be sure to be in this context.